improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -494,5 +494,36 @@ def test_dimension_preservation(self):
         self.assertListEqual(result_coord_order, expected_coord_order)
 
 
+class Test_alinfit(IrisTest):
+    """Test the alinfit method"""
+
+    def setUp(self):
+        """Creates X and Y data (Y is random)."""
+        array_size = 25
+        self.X = np.array(range(array_size))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [range built-in referenced when not iterating](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)